manim/renderer/opengl_renderer_window.py

```diff
@@ -28,6 +43,12 @@ def __init__(self, renderer, size=None, **kwargs):
 
         self.swap_buffers()
 
+        initial_position = self.find_initial_position(size, monitor)
+        print(initial_position)
+        print(size)
+        self.position = initial_position
```

Comment:
Did you forget to remove these?

Reply:
Whoops 😅. The second `self.position = initial_position` is actually a bug fix from manimgl where it wouldn't set the position sometimes with only one, but I haven't encountered it 🤔

Reply:
It works fine for me now. If anyone encountered an issue with the initial position, we can find more convenient solution.